pipeline/outputs/observe.md

```diff
@@ -0,0 +1,42 @@
+| http_user                  | Basic Auth Username                                                                                                                                                                                                                                                                                                                | ${OBSERVE_CUSTOMER} |
+| http_passwd                | Basic Auth Password. Requires http\_user to be set                                                                                                                                                                                                                                                                                 | ${OBSERVE_TOKEN} |
+| uri                        | Specify the HTTP URI for the Observe's data ingest                                                                                                                                                                                                                                                                                 | /v1/http/fluentbit |
+| format                     | The data format to be used in the HTTP request body                                                                                                                                                                                                                                                                                | msgpack   |
+| header                     | The specific header to instructs Observe how to decode incoming payloads                                                                                                                                                                                                                                                           | X-Observe-Decoder fluent |
+| compress                   | Set payload compression mechanism. Option available is 'gzip'                                                                                                                                                                                                                                                                      | gzip      |
+| tls.ca_file                | **For use with Windows**: provide path to root cert                                                                                                                                                                                                                                                                                |           |
+
+### Configuration File
+
+In your main configuration file, append the following _Input_ & _Output_ sections:
+
+```python
```

Comment:
It's not python :)

Reply:
Thanks for feedback, @patrick-stephens .  I removed `'''python` and committed my change.  

I'd also would also politely disagree with relegating Observe to a sub category under HTTP.  For example, based on your comment to inspect other plugins you can see Splunk's "plugin" is HTTP with some varying parameters, but several aligned with FluentBit's native HTTP.  

May I please kindly ask that Observe get a unique article in the Outputs page?

Reply:
Sure, it's not my call so will leave it with @agup006